src/service/wallet.tsx

```diff
@@ -1,3 +1,4 @@
+/** This service is a .tsx so it can present contentful toasts. */
```

Comment:
helpful comment. though I worry we'll put more and more JSX in the file. solution below.